### ISSUE TITLE
[Feature]: Prefill customer name

### ISSUE THREAD

**Issue by jamescrewdentials** [PaddleHQ/paddle-js-wrapper#11]
### Tell us about your feature request

It appears you cannot pass a customer name into Paddle checkout. I'd like to be able to prefill a customer name in the same way that I do the email address so that it appears in the dashboard with the other customer information once a checkout is completed.

### What problem are you looking to solve?

It is expected to be able to prefill basic customer data fields

### How important is this suggestion to you?

Important

**Comment by vijayasingam-paddle** [comment]
Hi @jamescrewdentials,
Thank you for reaching out to us.

This is a very good feedback, we are discussing about this internally and will get back to you once we have more information.

**Comment by bhupeshpr25** [comment]
Yes please, this is important in keeping track of users. Also the Customer name column being empty looks weird. Ideally should be filled with the card name used during checkout

**Comment by gillesvink** [comment]
I agree, this would greatly help. This is useful for sending the customer emails and having the data in the customer name column being entered correctly. Just one additional field on the checkout (maybe optionally activated) for the full name of the customer or business name to be passed to Paddle after checkout would fix this. Similar to how Stripe is handling that during checkout.
